docs/features/search/declarative-integration.md

```diff
@@ -0,0 +1,229 @@
+# Declarative Integrated Search Plugin
+
+> **Disclaimer:**
+> Declarative integration is in an experimental stage and is not recommended for production.
+
+This is a guide for experimenting with `Search` in a declarative integrated Backstage front-end application.
+
+## Main Concepts
+
+Using declarative integration, you can customize your Backstage instance without writing code, see this [RFC](https://github.com/backstage/backstage/issues/18372) for more information.
+
+In this new system architecture, everything that extends the core Backstage features is called a extension, so a extension could be since an api to a page component, which means all plugins can provide are extensions.
+
+Extensions produces outputs artifacts and these artifacts are inputs consumed by other extensions:
+
+![search extensions example](../../assets/search/search-extensions-example.svg)
+
+In the image above, a `SearchResultItem` extension outputs a component and this component is injected as input to the `SearchPage` "items" attachment point. The `SearchPage` in turn outputs a route path and element which are inputs attached to the `CoreRoutes` extension. Finally, the `CoreRoutes` renders the page element when the location matches the search path.
+
+The basic concepts briefly mentioned are crucial to understanding how the declarative version of the `Search` plugin works.
+
+## Search Plugin
+
+The search plugin is a collection of extensions that implement the search feature in Backstage.
+
+### Installation
+
+Only one step is required to start using the `Search` plugin within declarative integration, so all you have to do is to install the `@backstage/plugin-catalog` and `@backstage/plugin-search` packages, (e.g., [app-next](https://github.com/backstage/backstage/tree/master/packages/app-next)):
+
+```sh
+yarn add @backstage/plugin-catalog @backstage/plugin-search
+```
+
+The `Search` plugin depends on the `Catalog API`, that's is the reason we have to install the ` @backstage/plugin-catalog` package too.
```

Comment:
either that's or that is, not both

Reply:
Fixed 🙏🏻 : https://github.com/backstage/backstage/pull/20328/commits/8558c919cb0ee555b420cbc09d2a1de33be7917d